ui_testing/features/support/env.rb

```diff
@@ -0,0 +1,24 @@
+require 'capybara/cucumber'
+require 'capybara/dsl'
+require 'rspec'
+require 'pry'
+require_relative '../lib/codebar'
+
+
+# The below line of code now means that the chrome driver is registered before every run.
+# Any query stated at the top level of the env.rb file will be initiated at every cucumber run
+
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(app,:browser => :chrome)
+end
+
+Capybara.configure do |config|
+  config.ignore_hidden_elements = false #to ensure that all hidden elements on a page are recorded/available
+  config.default_max_wait_time= 10 #wait time for asynchronus processes to finsh
+  config.match = :prefer_exact #this setting is to ensure Capybara has specific matching rather than fuzzy logic
+  config.default_driver = :chrome # ensures chrome is the default driver
+  config.app_host = 'https://www.codebar.io' # provides the app host/core url
```

Comment:
This is testing against the live site. Is that your intention or should we uncomment the line below?

One benefit of automated testing is we can catch problems in code before it goes live